doc/release/README.bugs.rst

```diff
@@ -1,36 +1,34 @@
 .. _readme-bugs:
 
-=========================
-Reporting Bugs for Chapel
-=========================
-
-If you encounter a bug or problem using the Chapel compiler, please do
-not hesitate to contact the Chapel development team at the following
-address:
-
-  chapel-bugs@lists.sourceforge.net
-
-Note that mail sent to the above address will be publicly readable and
-that submitted test cases may be folded into our testing system unless
-you explicitly forbid it.  If you would prefer for your bug to be held
-in confidence by the Chapel team, please mail it to:
-
-  chapel_bugs@cray.com
-
-Please include as much information as possible in your email.  For
-example:
-
-  * the Chapel program that caused the problem (if possible and applicable)
-  * the behavior you are seeing
-  * the output of the ``$CHPL_HOME/util/printchplenv`` script
-  * the output of ``module list`` (when applicable)
-  * the version number of your Chapel compiler (``chpl --version``)
-  * the platform(s) you are running on and their OS versions
-  * the C/C++ compiler(s) you are using and their versions
-
-A list of unimplemented features and known bugs can be found in
-``$CHPL_HOME/STATUS``.  Feel free to check any unexpected behavior against
-this file.  Even if you run into a known issue, we encourage you to
-email us a bug report to make us aware that it's a problem for your
-code.  We are more than happy to suggest workarounds and/or to
-prioritize working on an issue that is impeding your progress.
+=============================
+Reporting Bugs against Chapel
```

Comment:
Yeah, or maybe even just "Reporting Bugs"?  I'll keep the Chapel in for now given that most of the siblings in the sidebar also mention the good language by name.